pubsub/gossipsub/gossipsub-v1.1.md

```diff
@@ -626,3 +627,28 @@ In order counter spam that elicits responses and consumes resources, some measur
   penalized by the score function. A peer transmitting lots of spam will quickly get graylisted,
   reducing the surface of spam-induced computation (eg validation). The application can take
   further steps and blacklist the peer if the spam persists after the negative score decays.
+
+### Recommendations for Network Operators
+
+An important issue to consider when deploying gossipsub is the peer discovery mechanism,
+which must provide a secure way of discovering new peers.
+Prior to gossipsub v1.1, operators were required to utilize an external peer discovery
+mechanism; with gossipsub v1.1 this is now entirely optional and the network can bootstrap
+purely through a small set of network entry points (bootstrappers) by utilizing Peer Exchange.
+
+In order to successfully bootstrap the network without a discovery service, network operators
+should
+- Create and operate a set of bootstrapper nodes, that are known within client software.
+- The bootstrappers should be configured without a mesh (ie set `D=Dlo=Dhi=Dout=0`)
```

Comment:
Can we add an option to the pubsub constructor in the go reference implementation that does this, e.g. `Bootstrapper()`.

Reply:
yes, we can do that.